integration/container/kill_test.go

```diff
@@ -31,6 +31,7 @@ func TestKillContainerInvalidSignal(t *testing.T) {
 }
 
 func TestKillContainer(t *testing.T) {
+	skip.If(t, testEnv.OSType == "windows") // TODO Windows: FIXME. No SIGWINCH
```

Comment:
Looks like you missed some; here and a couple more could be passed as third argument as well (just a nit, so fine for a follow-up)